### ISSUE TITLE
oldResource and newResource missing from EventDropArg typescript definition

### ISSUE THREAD

**Issue by hisuwh** [fullcalendar/fullcalendar#6010]
### Reduced Test Case

not reproducible with a link

### Bug Description

The `oldResource` and `newResource` properties (as described in the docs https://fullcalendar.io/docs/eventDrop) are missing from the Typescript definition for type `EventDropArg`.

### Screenshots

![image](https://user-images.githubusercontent.com/4470351/101024158-9efbb180-356b-11eb-9b5f-63ebdd842de6.png)

![image](https://user-images.githubusercontent.com/4470351/101024198-ade26400-356b-11eb-92f5-3f88c5a9fb87.png)

**Comment by lucianholt97** [comment]
Having the same issue... Any workarounds?

**Comment by hisuwh** [comment]
@lucianholt97 I've just defined my own interface for now and reference that instead:
```ts
import { EventDropArg } from "@fullcalendar/interaction";
import { ResourceApi } from "@fullcalendar/resource-common";

export interface ResourceEventDropArg extends EventDropArg  {
    oldResource?: ResourceApi;
    newResource?: ResourceApi;
}

```

**Comment by arshaw** [comment]
This has been fixed as of 5.5.0

I needed to make a breaking change to achieve this. EventDropArg now lives in `@fullcalendar/core` ...

```ts
import { EventDropArg } from "@fullcalendar/core";
```
